.script/tests/KqlvalidationsTests/CustomTables/imDNSGCPCloudDNS.json

```diff
@@ -0,0 +1,237 @@
+{
+    "name": "imDNSGCPCloudDNS",
```

Comment:
Please don't use imDNS for none ASIM tables or functions.

Reply:
fixed